CHANGELOG.md

```diff
@@ -1,10 +1,43 @@
-## vNext (TBD)
+## 12.0.0-alpha.0 (2023-03-21)
 
-### Deprecations
-* None
+This is the first pre-release of the next major version of our SDK.
+Please read more and discuss in the dedicated discussion: https://github.com/realm/realm-js/discussions/5416
+
+### Breaking changes
+
+Although this is a complete re-write of our SDK, we've strived to keep breakages to a minimum and expect our users to upgrade from v11 without any notible changes to their code-base.
```

Comment:
```suggestion
Although this is a complete rewrite of our SDK, we've strived to keep breakages to a minimum and expect our users to upgrade from v11 without any notable changes to their code-base.
```

However, is this really true? Aren't we listing some notable changes that users will need to make? Perhaps we should say "significant" rather than "notable"?